com.vrcfury.vrcfury/Editor/VF/Feature/FullControllerBuilder.cs

```diff
@@ -253,6 +255,27 @@ public class FullControllerBuilder : FeatureBuilder<FullController> {
             // (we do this after rewriting paths to ensure animator bindings all hit "")
             from.RewriteParameters(RewriteParamName);
 
+            var smoothedParams = model.smoothedPrms.ToDictionary(
+                x => x.name == "*" ? x.name : RewriteParamName(x.name), 
+                x => x);
+            var fromParams = from.parameters.ToDictionary(x => x.name, x => x);
+            from.RewriteParameters(param => 

```

Comment:
I was trying to make use of the `includeWrites: false` behavior here, if I combine with the call above I won't know if I do a write or not, unless I change the `RewriteParameters` method to include that information in the callback somehow. I agree having access to the original param name would be nice, especially since the smoothed layers wouldn't get doubled up `VF_` prefixes. 
How do you suggest keeping the `includeWrites: false` behavior and merging it with the call above?

Reply:
Fair enough. BTW: `includeWrites` was previously actually totally unused, and was introduced specifically to support this use case ;)